pantheon/src/main/java/tech/pegasys/pantheon/cli/subcommands/blocks/BlocksSubCommand.java

```diff
@@ -241,71 +241,127 @@ private MiningParameters getMiningParameters() {
         arity = "1..1")
     private final Long endBlock = null;
 
+    @Option(
+        names = "--format",
+        hidden = true,
+        description =
+            "The format to export, possible values are: ${COMPLETION-CANDIDATES} (default: ${DEFAULT-VALUE}).",
+        arity = "1..1")
+    private final BlockExportFormat format = BlockExportFormat.RLP;
+
     @Option(
         names = "--to",
+        required = true,
```

Comment:
So we don't currently support export to standard output correct?  An absent `--to` option is how I wold expect that to be configured.  It may be interesting for json export but it sounds wrong for RLP export.

Reply:
Right, no one wants to read binary data from std out :D But yeah, we might need to rethink it for json export.  Although given you can potentially export massive amounts of data, doesn't seem like the worst thing to require a file to export to.